nana/modules/meme.py

```diff
@@ -118,12 +120,12 @@ async def scam(client, message):
         return
 
 
-@app.on_message(Filters.user(AdminSettings) & Filters.command("shg", Command))
+@app.on_message(filters.user(AdminSettings) & filters.command("shg", Command))
 async def shg(_client, message):
-    await msg(message, text=random.choice(meme_strings.shgs))
+    await edrep(message, text=random.choice(meme_strings.shgs))
```

Comment:
Codacy found an issue: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)